Mathlib/Init.lean

```diff
@@ -11,7 +11,7 @@ import Mathlib.Tactic.Linter.UnusedTactic
 import Mathlib.Tactic.Linter.Style
 
 /-!
-This is the root file in Mathlib: it is imported by virtually *all* Mathlib files.
+This is the root file in Mathlib: it is imported by *all* Mathlib files.
```

Comment:
Wasn't it more correct with "virtually"?  There are still some mathlib files that do not import `Mathlib.Init`, after all.

Reply:
I don't feel strongly about this; reverted to the previous phrasing.